easybuild/easyconfigs/f/fontconfig/fontconfig-2.12.6-foss-2017b-libpng-1.6.34.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'fontconfig'
+version = '2.12.6'
+versionsuffix = '-libpng-1.6.34'
+
+homepage = 'http://www.freedesktop.org/software/fontconfig'
+description = """Fontconfig is a library designed to provide system-wide font configuration, customization and
+application access."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = ['http://www.freedesktop.org/software/fontconfig/release/']
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['064b9ebf060c9e77011733ac9dc0e2ce92870b574cca2405e11f5353a683c334']  # fontconfig-2.12.6.tar.gz
+dependencies = [
+    ('expat', '2.2.5'),
+    ('freetype', '2.8.1', versionsuffix),
+]
+
+# use same binutils version that was used when building GCCcore toolchain
+builddependencies = [('pkg-config', '0.29.2'), ('gperf', '3.1')]
```

Comment:
dependencies should be listed one per line

also, the comment above this line is incorrect, binutils is not listed here